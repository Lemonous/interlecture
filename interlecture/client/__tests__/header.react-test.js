import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../core/header';

function onSubmit(event) {
  event.preventdefault();
  console.log('onSubmit');
}

global.window = {}
window.django2react = {}
window.django2react.username = undefined
window.django2react.firstname = undefined
window.django2react.lastname = undefined

test('Login screen looks like it should', () => {
  const component = renderer.create(
    <Header
      user={window.django2react.username}
      firstname={window.django2react.firstname}
      lastname={window.django2react.lastname}
    />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
