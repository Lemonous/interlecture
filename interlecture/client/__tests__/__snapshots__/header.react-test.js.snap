// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login screen looks like it should 1`] = `
<header
  style={
    Object {
      "background": "linear-gradient(to bottom, rgba(0, 0, 0, 255), rgba(0, 0, 0, 0))",
      "height": "200px",
      "left": 0,
      "position": "absolute",
      "top": 0,
      "width": "100%",
      "zIndex": 5,
    }
  }
>
  <div
    style={
      Object {
        "color": "white",
        "padding": "5px 10px",
        "textAlign": "right",
        "width": "100%",
      }
    }
  >
    <p>
      Â 
    </p>
    <div
      style={
        Object {
          "color": "white",
          "fontFamily": "Raleway, sans-serif",
          "fontSize": "50pt",
          "fontWeight": "300",
          "marginTop": "50px",
          "textAlign": "center",
          "width": "100%",
        }
      }
    >
      INTERLECTURE
    </div>
  </div>
</header>
`;
